benchmarks/ekomark/benchmark/external/lhapdf_utils.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+
+from eko import basis_rotation as br
+
+
+def compute_LHAPDF_data(theory, operators, pdf, rotate_to_evolution_basis=False):
+    """
+    Run LHAPDF to compute operators.
+
+    Parameters
+    ----------
+        theory : dict
+            theory card
+        operators : dict
+            operators card
+        pdf : lhapdf_type
+            pdf 
+        rotate_to_evolution_basis: bool 
+            rotate to evolution basis
+    
+    Returns
+    -------
+        ref : dict
+            output containing: target_xgrid, values, skip_pdfs
+    """
+
+    target_xgrid = operators["interpolation_xgrid"]
+    skip_pdfs = [22, -6, -5, 5, 6]
+
+    out_tabs = {}
+    for pid in br.flavor_basis_pids:
+
+        if pid in skip_pdfs:
+            continue
+
+        # collect lhapdf
+        me = []
+        for x in target_xgrid:
+            xf = pdf.xfxQ2(pid, x, operators["Q2grid"][0])
```

Comment:
Here you are hardcoding `operators["Q2grid"][0]` as we were doing, but in principle the `[0]` is there because it is a list and may contain more than a single element.
If more than one element is present all of them should be benchmarked, and since `lhapdf` can do it this should be tested (eko it's somehow optimizing for computing multiple `Q2` at once, so the test it's meaningful).

Reply:
Yes, same for apfel, when everything is fine with one q2, I'll add a loop.

Reply:
:ok_hand: